chia/full_node/mempool_check_conditions.py

```diff
@@ -292,8 +293,91 @@ def parse_condition(cond: SExp, safe_mode: bool) -> Tuple[int, Optional[Conditio
     return cost, cvl
 
 
-def get_name_puzzle_conditions(
+def get_name_puzzle_conditions_python(
```

Comment:
this particular part of the patch is much clearer when disabling whitespace changes, because the python version of this function is essentially unchanged, and match up correctly